cogboard-webapp/src/hooks/index.js

```diff
@@ -42,7 +52,38 @@ export const useFormData = (data) => {
     setFieldValue(fieldName, fieldValue);
   };
 
-  return { values, handleChange };
+  const validateField = (fieldName, fieldsValues) => {
+    if (status.submited || status.onChange) {
+      validationSchema.validateAt(fieldName, fieldsValues, {abortEarly: false})
+        .then(() => {
+          if (fieldName in errors) {
+            const errorsTmp = {...errors}
+            delete errorsTmp[fieldName]
+            setErrors(errorsTmp)
+          }
+        }).catch(e => {
```

Comment:
one liner and use error as parameter name

Reply:
done